pkgs/by-name/co/coredns/package.nix

```
@@ -53,7 +34,35 @@ buildGoModule rec {
 
   # Override the go-modules fetcher derivation to fetch plugins
   modBuildPhase = ''
-    for plugin in ${builtins.toString (attrsToPlugins externalPlugins)}; do echo $plugin >> plugin.cfg; done
+    cp plugin.cfg plugin.cfg.orig
+    ${
+      (lib.concatMapStringsSep "\n" (
+        plugin:
+        let
+          position = plugin.position or "end-of-file";
+          formatPlugin = { name, repo, ... }: "${name}:${repo}";
+        in
+        if position == "end-of-file" then
+          "echo ${formatPlugin plugin} >> plugin.cfg"
```

```suggestion
          "echo '${formatPlugin plugin}' >> plugin.cfg"
```
not sure if this would be a problem in practice